js/modules/loki_rpc.js

```diff
@@ -477,14 +512,6 @@ const lokiFetch = async (url, options = {}, targetNode = null) => {
         const thisIdx = onionReqIdx;
         onionReqIdx += 1;
 
-        log.debug(
```

Comment:
Is this just gone now? It is on `debug` so I think we could keep it.

Reply:
yes, it's just creates a lot of noise. It's not an indication of a problem. We send a lot of requests over ORs. And now we just include this path on any error/exception